### PR TITLE
Fix crash report in options dialog. Argument out of range error when …

### PR DESCRIPTION
**Pull request by rrelyea**
…setting selectedIndex from a mouseUp handler.

Fixes https://github.com/NuGet/Home/issues/2359

### CODE REVIEW

**Comment by deepakaravindr** [comment]
:ship:

**Comment by joelverhagen** [comment]
:shipit:

**Comment by rrelyea** [comment]
Merged into dev and release-3.4.0

File: src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs [hunk 1]
```diff
@@ -471,7 +471,11 @@ private void PackageSourcesListBox_MouseUp(object sender, MouseEventArgs e)
             var currentListBox = (ListBox)sender;
             if (e.Button == MouseButtons.Right)
             {
-                currentListBox.SelectedIndex = currentListBox.IndexFromPoint(e.Location);
+                int itemIndexToSelect = currentListBox.IndexFromPoint(e.Location);
```
Comment (yishaigalatzer) [inline src/NuGet.Clients/Options/PackageSourcesOptionsControl.cs:474]: var

**Pull request closed without merging**